rose/web/game.js

```diff
@@ -44,7 +44,7 @@ var ROSE = (function() {
         var state = msg.payload;
 
         // Update
-        this.controller.update(state.started);
+        this.controller.update(state.started, state.players.length);
```

Comment:
Better pass entire state, so we can have a uniform update(state) - will allow later to replace this code with a loop calling update on all components.

Reply:
Done